concepticondata/conceptlists.tsv

```diff
@@ -408,4 +408,4 @@ KochGrunberg-1914-821	Koch-Grünberg, Theodor	 1914		821	annotated, areal, basic
 BarbosaEscobar-2021-12	Barbosa Escobar, Francisco and Velasco, Carlos and Motoki, Kosuke and Byrne, Derek Victor and Wang, Qian Janice	2021	  	 12	specific	English	English, Chinese, Japanese, Spanish	https://doi.org/10.1371/journal.pone.0252408	BarbosaEscobar2021		The list includes 12 emotion adjectives arranged in form of a wheel. The adjectives were based on the valence x arousal circumplex-inspired emotion questionnaire (CEQ, [Jaeger et al.](:bib:Jaeger2021)) and were translated from English into Mandarin Chinese, Japanese, and Spanish.		              
 Epps-2021-843	Van Epps, Briana, and Carling, Gerd and Sapir, Yair	2021		843	documentation, historical, norms	English	Old Norse, Norwegian, Old Swedish, Swedish, Jamtlandic, Elfdalian	https://github.com/gerdcarling/gendernorthscandinavian	Epps2021		This list is derived from of a study focusing on lexical gender assignment in six North Scandinavian varieties. It provides statistical information about semantic change in about 1300 cognate sets and their respective patterns of change from their ancestral forms in Old Norse. Since the data is arragned by cognate sets, identical elicitation glosses for concepts are distinguished by adding numerals to the end of the respective glosses. These are retained in the column LEXIBANK_GLOSS, but ignored in our representation.	Appendix A	
 Nordenskioeld-1905-50	Nordenskiöld, Erland	1905		50	basic, areal	German	South American languages	https://www.pueblos-originarios.ucb.edu.bo/Record/106001946?lng=da	Nordenskioeld1905	Nordenskioeld1905	This list compares five languages of the Madre de Dios region in the Peruvian selva across 50 basic concepts. While four languages are part of the hypothesized Pano-Takana language family, one language is part of the Harákmbut family. Of the Pano-Takana languages, two are unclassified with respect to the internal classification of the family. The data is based on fieldwork from the original author which was done in 1904 and 1905.	275-276	
-
+RonaTas-2011-431	Róna-Tas, András, and Berta, Árpád	2011		431	historical	English	Hungarian, Old Hungarian, Early Ancient Hungarian, West Old Turkic		RonaTas2011		This list features 431 Hungarian words that were loaned from West Old Turkic, compiled by András Róna-Tas.
```

Comment:
I think the comma after "András" has to go in order to be correctly in the bibtex format, right?